### DISCUSSION TITLE
agregar opcion general de pesos a pesos

### DISCUSSION THREAD

**Discussion by fedethxen** [IvanMass99/Calculadora-Gamer#4] [Ideas]
Seria como calculas para steam, pero se da el caso que para nintendo switch tenes las dos opciones: de dolares a pesos y de pesos a pesos... o bien se podría agregar un boton  para filtrar de pesos a pesos o dolares a pesos en este caso te mostraria solo las plataformas con esas opciones, otra opcion es agregar un botoncito dentro de cada componente de market para elegir el tipo de calculo. Muy buena la app, te felicito.... si queres podriamos agregarla a un hosting con dominio propio

**Comment by IvanMass99** [comment]
¡Hola Fede! Gracias por tu sugerencia, la voy a tener en cuenta para las siguientes actualizaciones. Con respecto al hosting, de momento no genero las suficientes ganancias con el proyecto para poder costearlo. Saludos!
